Tests/test_content.py

```diff
@@ -496,10 +496,19 @@ def create_result_files(tests_data_keeper):
         skipped_integrations_file.write('\n'.join(skipped_integration))
 
 
+def change_placeholders_to_values(changes_map, config_item):
+    item_as_string = json.dumps(config_item)
+    for key, value in changes_map.items():
+        item_as_string = item_as_string.replace(key, value)
+    return json.loads(item_as_string)
+
+
 def set_integration_params(demisto_api_key, integrations, secret_params, instance_names, playbook_id,
-                           prints_manager, thread_index=0):
+                           prints_manager, server_host, thread_index=0):
     for integration in integrations:
-        integration_params = [item for item in secret_params if item['name'] == integration['name']]
+        changes_map = {'%%SERVER_HOST%%': server_host}
```

Comment:
better check if this param is there and only then do it

Reply:
Resolved after the slack conversation